src/global/token.js

```diff
@@ -1,11 +1,13 @@
+import Cookies from 'js-cookie'
+
 init()
 
 export default {
   key: () => get('key'),
   authzHeader: () => `${get('token_type')} ${apiToken()}`,
   reset,
   accessToken: () => get('access_token'),
-  setAccessToken: (t) => sessionStorage.setItem('access_token', t),
+  setAccessToken: (t) => Cookies.set('access_token', t, { expires: 7, path: '/' }),
```

Comment:
```suggestion
  setAccessToken: (t) => Cookies.set('access_token', t, { expires: 7, path: '/',  sameSite: 'strict' }),
```
As long as this doesn't prevent api and ui from talking. It works on cover